asati_ecr.tf

```diff
@@ -0,0 +1,66 @@
+# ArchiveSpace AirTable Integration (asati) containers
+# This is a standard ECR for an ECS with a Fargate launch type
+locals {
+  ecr_asati = "asati-${var.environment}"
+}
+module "ecr_asati" {
+  source            = "./modules/ecr"
+  repo_name         = "docker-asati"
```

Comment:
You should probably just call this `asati` -- we have only used the `docker-` prefix when we are using a 3rd party docker application that is just getting some minor customization (like we did with Matomo). Any DLS-developed applications shouldn't have `docker-` in their name.